src/css/support.js

```diff
@@ -46,61 +46,57 @@ define([
 		documentElement.removeChild( container );
 	}
 
-	// Support: node.js jsdom
-	// Don't assume that getComputedStyle is a property of the global object
```

Comment:
I'm not sure if it comes into play here, but `window` may be a non-Window global object: https://github.com/mzgol/jquery/blob/jsdom-update/src/intro.js#L38

In any case, the comment probably does need rewriting.